build/references.py

```diff
@@ -169,7 +169,10 @@ def get_standard_citatation(citation, cache, override):
 converted_text = semicolon_separate_references(converted_text)
 
 # Template using jina2
-template = jinja2.Template(converted_text)
+#env = jinja2.Environment(comment_start_string='{##')
+#template = env.get_template(converted_text)
+template = jinja2.Environment(loader=jinja2.BaseLoader(),
+                              comment_start_string='{##').from_string(converted_text)
```

Comment:
For readability, I think it's best to have `.from_string(converted_text)` or `template = env.get_template(converted_text)` on its own line.

Also remove commented out code.